app/resources/add_note_base.html.template

```diff
@@ -89,6 +104,14 @@
         <div class="end-multi-columns"></div>
       </div>
 
+      <a href="#" class="status-button" style="text-decoration:none">
```

Comment:
i put text-decoration:none in order to remove the underline of the text in the button. Already moved it to css-default.css :)